nodejs/overview/msteams-client.md

```diff
@@ -1,61 +1,111 @@
 ---
-title: Microsoft Teams JavaScript client library
+title: Microsoft Teams Client API reference
 description: Reference documentation for the latest Microsoft Teams JavaScript client library
-keywords: msteams teams client sdk javascript library reference latest
+ms.date: 01/03/2023
+ms.topic: reference
+keywords: msteams live-share teamsfx teams client sdk javascript library reference latest
 ---
-# Microsoft Teams JavaScript client library
+# Microsoft Teams Client API reference
 
-Starting with *@microsoft/teams-js* version `2.0.0`, the Microsoft Teams JavaScript client library has been refactored to enable [Teams apps to run in Outlook and Office](/microsoftteams/platform/m365-apps/overview), in addition to Microsoft Teams. From a functional perspective, the TeamsJS SDK is a superset of TeamsJS 1.x versions: it supports existing Teams app functionality while adding the ability to host Teams apps in Outlook and Office.
+This page contains general information about the client library to support you in the development of using The Microsoft Teams Client SDK.
 
-> [!TIP]
-> Refer to the [Teams JavaScript client library](/microsoftteams/platform/tabs/how-to/using-teams-client-library) overview for guidance on building both Teams-only and Microsoft 365-enabled apps and for current SDK capability support across Teams, Outlook, and Office.
+| Library | Description | APIs |
```

Comment:
The "APIs" column label feels a little misleading, like it links to the actual code rather than reference documentation.

Reply:
would 'Code' be a better title?

Reply:
If it's linking to the reference docs, maybe just say "Docs" or "Reference"

Reply:
Also, maybe just me, but I'd almost consider turning this into an intro paragraph instead. Set the stage for why these are grouped and just have the links be off the library name keyword. Can help with suggestion if you'd like to talk about how to do that. Then the following sections can be more details about the individual library.

Reply:
For column label, how about "API reference"?

Also agree with Josh on the description--would be great to flesh it out into a full paragraph describing the different libraries that comprise Teams SDK.